roles/lnls-ans-role-java/vars/Debian-buster.yml

```diff
@@ -1,8 +1,9 @@
 ---
-# JDK version options include:
-#   - java
-#   - openjdk-8-jdk
-__java_packages:
-  - openjdk-11-jdk
-  - openjdk-11-jre
-  - openjfx
+  # JDK version options include:
+  #   - java
+  #   - openjdk-8-jdk
+  __java_packages:
```

Comment:
The `__java_packages` seems wrongly indented

Reply:
fixed